README.md

```diff
@@ -46,15 +39,15 @@ and drop it on your lxc templates path (usually `/usr/share/lxc/templates`)._
 ```sh
 git clone https://github.com/fgrehm/vagrant-lxc-base-boxes.git
 cd vagrant-lxc-base-boxes
-make precise
+make DEBIAN_BOXES=jessie debian
 ```
 
 By default no provisioning tools will be included but you can pick the ones
 you want by providing some environmental variables. For example:
 
 ```sh
 PUPPET=1 CHEF=1 SALT=1 BABUSHKA=1 \
-make precise
+make UBUNTU_BOXES=precise ubuntu
```

Comment:
This hunk seems unrelated to the actual commit of removing the 'unmaintained' notice from the README. Also I am not sure this change is appropriate.